app/src/main/res/layout/spotlight_fragment_test_activity.xml

```diff
@@ -15,7 +15,7 @@
       android:id="@+id/sample_spotlight_target"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:text="This is a sample text view"
+      android:text="@string/sample_text_view"
```

Comment:
```suggestion
      android:text="@string/sample_text_view_text"
```

Reply:
Renamed the string resource 'sample_text_view' to 'sample_text_view_text'